tests/lib/rules/require-expect.js

```diff
@@ -275,6 +275,16 @@ ruleTester.run("require-expect", rule, {
             options: ["except-simple"]
         },
 
+        // Multiple assert statements only report one error per test
+        {
+            code: "test('name', function(assert) { maybe(function() { assert.ok(true); assert.ok(true); }) });",
+            errors: [{
+                message: exceptSimpleErrorMessage("assert.expect"),
+                type: "CallExpression"
+            }],
+            options: ["except-simple"]
+        },
```

Comment:
I was about to link to this section in [Working with Rules](http://eslint.org/docs/developer-guide/working-with-rules#invalid-code), but I see that line/column aren't mentioned there. Oops! That should definitely be added. Anyway, you might want to look at the RuleTester source so you know for sure how "column" is spelled (column, col, etc.). I believe it's 1-based but don't quote me.